tests/test.py

```diff
@@ -96,6 +103,12 @@ def main():
         "Pi is roughly 3",
         py_file_path=py_file_path)
 
+    # TO DO: enable R test when R is enabled in Spark (2.1)
```

Comment:
remove space between "TO" and "DO".  People often search code bases for "TODO", and would miss this.